stackrox/base/policy/scanner-db.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# Source: stackrox-central-services/templates/02-scanner-05-network-policy.yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
move all `networkpolicies` to a separate directory under `base` called `networkpolicies`